download_action.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+)
+
+type DownloadAction struct {
+	BaseAction `yaml:",inline"`
+	Url        string // URL for downloading
+	Filename   string // File name, overrides the name from URL.
+	Forced     bool   `yaml:"force"` // Forced re-download of URL
```

Comment:
Drop the forcing, unless there is a strong use-case for it. In a recipe that would essentially mean always-redownload, which when caching is added would be a cache workaround. For one-time forcing the user can always drop the cache